.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,298 @@
+eclipse.preferences.version=1

```

Comment:
It seems like you commit the .gitignore along with .setting/*, you had to send it on a commit first :)

Can you please delete these three files?